src/v2i-hub/CDASimAdapter/scripts/send_sim_detected_object_udp.py

```diff
@@ -0,0 +1,89 @@
+
+import socket
+import sys
+import json
+import time
+
+# Script for integration testing CDASimAdapter Time Sync functionality.
+# This python script sends periodic time sync messages to a configurable
+# host and port. To Test the Time Sync functionality of the CDASimAdapter
+# set port to the value of the TIME_SYNC_PORT environment variable.
+#
+# TODO Move this script into a more permanent location
+count_num = 0
+
+def generate_sim_external_object():
+    jsonResult = {
+        "metadata": {
+            "type": "Application",
+            "subtype": "SensorDetectedObject",
+            "timestamp": 123,
+            "isSimulated": True
+        },
+        "payload": {
+            "sensor": {
+            "id": "SomeID",
+            "type": "SematicLidar",
```

Comment:
We talked about removing sensor type and location from this message. The message should now only include sensor id and projection string.

Reply:
> We talked about removing sensor type and location from this message. The message should now only include sensor id and projection string.

addressed

Reply:
I guess technically we do not need it for sensor registration. We definitely need it for sensor detected object.

Reply:
How do we know the location (x,y,z) of the sensor if no proj_string is applied to sensor?